client/state/sites/selectors/uses-wp-admin-interface.ts

```diff
@@ -0,0 +1,6 @@
+import getSiteOption from './get-site-option';
+import type { AppState } from 'calypso/types';
+
+export default function usesWPAdminInterface( state: AppState, siteId?: number | null ) {
```

Comment:
It would be better to use the `get`-prefix instead of the `use`-prefix as it's the prefix of the hook 🙈